api/src/prescription/learner-management/application/jobs/import-organization-learners-job-controller.js

```diff
@@ -15,7 +16,13 @@ class ImportOrganizationLearnersJobController extends JobController {
   async handle({ data }) {
     const { organizationImportId } = data;
 
-    return usecases.addOrUpdateOrganizationLearners({ organizationImportId });
+    try {
+      return usecases.addOrUpdateOrganizationLearners({ organizationImportId });
+    } catch (err) {
+      if ((!err.prototype) instanceof DomainError) {
+        throw err;
+      }
```

Comment:
Normalement vous pouvez l'écrire de cette façon:
```suggestion
      if (!err instanceof DomainError) {
        throw err;
      }
```

Reply:
Je suis étonné que le test passe d'ailleurs ^^ car quand j'ai testé cette écriture dans la console node: `(!err.prototype) instanceof DomainError` et je n'arrive pas à la faire fonctionner. 😅

Reply:
J'ai changé, merci 👍 

Le test était effectivement pas bon 🙈